src/order-items/order-items.service.ts

```diff
@@ -41,13 +39,70 @@ export class OrderItemsService {
     }
 
     async findOne(id: number) {
+        const data = await this.prisma.orderItem.findUnique({
+            where: { id: id },
+            select: {
+                workflow: {
+                    select: {
+                        id: true,
+                        name: true,
+                        sequence: true,
+                    }
+                }
+            }
+        });
 
-        return await this.prisma.orderItem.findUnique({
-            where: {
-                id: id
+        if (!data) {
+            throw new NotFoundException('Invalid Order Id');
+        }
+
+        const sequence = Array.isArray(data.workflow?.sequence) ? data.workflow.sequence : [];
```

Comment:
Why is this optional is required here? Verify without the optional and provide the reason for it's use

Reply:
I skipped in the mind, why did use this optional?